### ISSUE TITLE
Provide LICENSE

### ISSUE THREAD

**Issue by rlaemmel** [101companies/101simplejava#9]
At this point we generally seem to assume the following license, but this could be changed, if needed.

http://101companies.org/wiki/101companies:License

Should the LICENSE file hence at the top of the repo? Is just a pointer enough? Say, what is the smallest footprint for addressing the LICENSE duty?

**Comment by mmonschau** [comment]
101repo is now under MIT-License (https://github.com/101companies/101repo/blob/master/LICENSE)
And https://101wiki.softlang.org/101license still does not exist although its linked in 101project ( https://101wiki.softlang.org/Concept:101project )

**Comment by kevin-klein** [comment]
101project page is retired and 101license has no meaning as far as i know.

**Comment by kevin-klein** [comment]
101code is usually licensed like this: https://github.com/101companies/101repo/blob/master/LICENSE

**Comment by mmonschau** [comment]
then provide LICENSE-files

**Comment by kevin-klein** [comment]
what do you mean? i think we just forgot the 101haskell, 101java etc repos.

**Comment by mmonschau** [comment]
each repo needs a license file

**Comment by kevin-klein** [comment]
yes, we should probably take care of this. @rlaemmel i can find the following (semi-used) repos without license files:
https://github.com/101companies/101worker
https://github.com/101companies/101simplejava
https://github.com/101companies/101courses
https://github.com/101companies/101haskell

should i put them under the same license as 101repo?
